tests/python/relay/test_pass_partition_graph.py

```diff
@@ -989,7 +989,7 @@ def test_partition():
         # conv + relu, conv + relu -> no fusion, 4 partition each with a single op
         test_detect_pattern([conv2d_bias_relu_pat], False, False, 4)
         # conv + bn + sigmoid + relu, conv + sigmoid + relu -> no fusion
-        test_detect_pattern([conv2d_bias_relu_pat, conv2d_relu_pat], True, True, 5)
+        test_detect_pattern([conv2d_bias_relu_pat, conv2d_relu_pat], True, True, 7)
```

Comment:
Would it make sense to add a few tests here? You've done a great job with tests that confirm the correctness of graph partitioned for DNNL, but havent tested that the new patterns are properly fused. All your new tests would pass even if none of the new pattern matches were offloaded as far as I can tell.

Reply:
Sure, more tests are added.